components/tests/ui/resources/robot.template

```diff
@@ -20,7 +20,7 @@ ${SERVER_ID}            1
 ${BROWSER}              Firefox
 ${DELAY}                0
 ${WAIT}                 50
-${TIMEOUT}              1
+${TIMEOUT}              60
```

Comment:
It was reported by @will-moore and fixed in develop in https://github.com/will-moore/openmicroscopy/commit/4daa4f9b23dc7cf274c6c99ec9f473a8fe252af8